Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -825,6 +825,12 @@ theorem Integrable.measure_ge_lt_top {f : α → β} (hf : Integrable f μ) {ε
     (memℒp_one_iff_integrable.2 hf).snorm_ne_top
 #align measure_theory.integrable.measure_ge_lt_top MeasureTheory.Integrable.measure_ge_lt_top
 
+/-- A non-quantitative version of Markov inequality for integrable functions: the measure of points
+where `‖f x‖ > ε` is finite for all positive `ε`. -/
+lemma Integrable.measure_gt_lt_top {f : α → β} (hf : Integrable f μ) {ε : ℝ} (hε : 0 < ε) :
```

Comment:
I now found the existing result `Integrable.measure_norm_ge_lt_top`, which was essentially identical to one of the lemmas I added.

I am tempted to rename this existing result and the `gt` counterpart here as `Integrable.measure_norm_ge_lt_top` and `Integrable.measure_norm_gt_lt_top`, respectively. Then the new ones, used in the layercake file, could be called `Integrable.measure_gt_lt_top`, `Integrable.measure_ge_lt_top` etc. The justification for the naming would be that the new ones have the `gt`/`ge` on values, whereas the old one has it for the norm of the value.

Is it ok to rename?

Reply:
Yes, the rename makes perfect sense. Maybe move the other lemmas `Integrable.measure_ge_lt_top` and so on close to this one, for easier discoverability?